lib/index.js

```diff
@@ -132,6 +134,43 @@ const invokeHandler = async function invokeHandler(options) {
   logger.info(response.body);
 };
 
+/**
+ * Run performance test on deployed function
+ *
+ * @param {object} options - Command line options.
+ */
+const perfHandler = async function perfHandler(options) {
+  logger.info(`Running performance test on function ${options.function}.
+Executing ${options.maxRequests} invocations with ${options.concurrency} "thread${options.concurrency > 1 ? 's' : ''}".
+Stand by for results...
+`);
+  const report = await perf(options.function, options.maxRequests, options.concurrency);
+  logger.info('Perf summary');
+  logger.info('============');
+
+  logger.info(columnify({
+    'Total time': `${report.totalTimeSeconds.toFixed(1)} ms`,
```

Comment:
I don't understand the units here: `totalTimeSeconds` is seconds, the suffix `ms` is milliseconds?

Reply:
:) fixed.